### PR TITLE
[chaoss] Add new panel for Pull Request Merge Duration

### PR DESCRIPTION
**Pull request by anajsana**
I've added a pull request merge duration new panel with its documentation (markdown) and a screenshot

### CODE REVIEW

File: docs/_chaoss-gmd-cde/pull_request_merge_duration.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+The main idea behind this panel is to help us identify a project state (growth maturity or decline) by answering custom queries like:
+* **What is the duration of time between code merge request and code commit?**
+* **What organizations/authors are being more efficient when closing a pull request?**
+* **Is my project taking too long when closing code merge requests?**
+
+
+### What should I look for in the panel?
+
+You can see if a pull request is taking too long to be merged by looking at gauge visualization. You can set up time ranges to help you visualize different scenarions (e.g fast, medium or slow response)
```
Comment (alpgarcia) [inline docs/_chaoss-gmd-cde/pull_request_merge_duration.md]: Not sure if this is true. I'd say we can visualize time to close in that specific time frame. For instance, how good or bad we were closing pull request created last year, two years ago or last week.

File: docs/_chaoss-gmd-cde/pull_request_merge_duration.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+created_at: 2019-01-23
+grimoirelab_version: 0.2.3
+layout: panel
+---
+
+
+This panel focuses on **Pull request merge duration**.
+
+
+This panel focuses on pull request merge duration, defined by the time between a code merge request and code commit. In this case, we focus on Github pull request. When we measure Github, we cannot identify if a pull request has been merged, only if is open or closed. That's the reason why we will based our panel on pull request closing time
```
Comment (alpgarcia) [inline docs/_chaoss-gmd-cde/pull_request_merge_duration.md]: [...] we focus on **GitHub** pull **requests**.

File: docs/_chaoss-gmd-cde/pull_request_merge_duration.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+You can see if a pull request is taking too long to be merged by looking at gauge visualization. You can set up time ranges to help you visualize different scenarions (e.g fast, medium or slow response)
+
+Moreover, you can filter pull request merge duration by author or organization to compare different resuts and see how many pull requests have been merged over a period of time
+
+### List of metrics provided on the panel
+* **Gauge visualization**: median time to close pull requests over a period of time
+* **Organizarion list based on closed pull requests**: time to close pull request filtered by organization name and number of authors involve on each organization
+* **Number of pull requests**: Amount of pull requests over a period of time
+* **Median time open days**: median time to close pull request during the selected time range
+
```
Comment (alpgarcia) [inline docs/_chaoss-gmd-cde/pull_request_merge_duration.md]: Please use dots at the end of each sentence.

File: docs/_chaoss-gmd-cde/pull_request_merge_duration.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+### What should I look for in the panel?
+
+You can see if a pull request is taking too long to be merged by looking at gauge visualization. You can set up time ranges to help you visualize different scenarions (e.g fast, medium or slow response)
+
+Moreover, you can filter pull request merge duration by author or organization to compare different resuts and see how many pull requests have been merged over a period of time
+
+### List of metrics provided on the panel
+* **Gauge visualization**: median time to close pull requests over a period of time
+* **Organizarion list based on closed pull requests**: time to close pull request filtered by organization name and number of authors involve on each organization
```
Comment (alpgarcia) [inline docs/_chaoss-gmd-cde/pull_request_merge_duration.md]: * Typo in **Organization**.
* ...pull **requests** filtered...
* ...authors **involved**...

**Comment by anajsana** [comment]
Thank you @alpgarcia ! I've just updated the panel and the documentation according to your feedback

**Comment by alpgarcia** [comment]
Thanks @anajsana, markdown widget needs some small changes:
  * `we focus on GitHub pull request.` should be `requests.`
  * Following sentence in the markdown widget is mixing future and past ;):
`That's the reason why we will based our panel on pull request closing time.`

I'd also use capital letters in `Big Numbers` and `Time to close days` metric custom labels

**Comment by alpgarcia** [comment]
Thanks!

**Pull request closed without merging**